### PR TITLE
Ft create reusable add button component

### PR DESCRIPTION
**Pull request by MutegekiHenry**
#### What does this PR do?
Creates reusable button for creating resources
#### Description of Task to be completed?

- Create a resuable button for adding new resources such as namespaces, projects, deployments 

#### How should this be manually tested?
- Running the branch locally

#### Screenshots (if appropriate)
CreateButton

### CODE REVIEW

File: src/components/ButtonComponent/ButtonComponent.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ width: 56px;
+ height: 56px;
+ background-color: #F7B21F;
+ border-radius: 50%;
+ border: none;
+ box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+
+.RoundAddButton:focus {
```
Comment (Mubangizi) [inline src/components/ButtonComponent/ButtonComponent.css]: The button should remove the shadow hover. As of now its not doing anything
```suggestion
.RoundAddButton:hover {
```

File: src/components/ButtonComponent/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {ReactComponent as ButtonPlus} from '../../assets/images/buttonplus.svg'
```
Comment (Mubangizi) [inline src/components/ButtonComponent/index.js]: Imports need to be terminated
```suggestion
import {ReactComponent as ButtonPlus} from '../../assets/images/buttonplus.svg';
```

File: src/components/ButtonComponent/index.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+import './ButtonComponent.css'
+
```
Comment (Mubangizi) [inline src/components/ButtonComponent/index.js]: From line 5 - 9 please implement 2-space indenting

File: src/components/ButtonComponent/ButtonComponent.css (hunk 1 above)
Comment (StephenAraka) [reply src/components/ButtonComponent/ButtonComponent.css]: Hey @Mubangizi Allan, this is not a mistake.
The effect occurs on click, not on hover.

**Comment by MutegekiHenry** [comment]
@Mubangizi i have worked on the changes

File: .eslintrc.json [hunk 4]
```diff
@@ -1,48 +1,26 @@
...
     "env": {
-      "browser": true,
-      "es6": true
+        "browser": true,
+        "es6": true
     },
     "extends": [
-      "eslint:recommended",
-      "plugin:react/recommended"
+        "plugin:react/recommended",
```
Comment (Mubangizi) [inline .eslintrc.json]:
```suggestion
        "eslint:recommended",
```

File: src/components/ButtonComponent/index.js [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
+import './ButtonComponent.css';
 
- const CreateButton =(props) =>( 
+  const CreateButton =(props) =>( 
       <button className="RoundAddButton" onClick={props.onClick}>
-        < ButtonPlus/>
+          < ButtonPlus/>
       </button>
-   );
+    );
```
Comment (Mubangizi) [inline src/components/ButtonComponent/index.js]:
```suggestion
  );
```

**Pull request merged**